### PR TITLE
pythonPackages.fastparquet: init at 0.2.1

### PR DESCRIPTION
**Pull request by veprbl**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by veprbl** [comment]
@GrahamcOfBorg build pythonPackages.fastparquet

**Comment by veprbl** [comment]
@GrahamcOfBorg build python3Packages.fastparquet

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/fastparquet/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, isPy3k, fetchPypi, numba, numpy, pandas,
+pytestrunner, thrift }:
+
+buildPythonPackage rec {
+  pname = "fastparquet";
+  version = "0.2.1";
+
+  disabled = !isPy3k;
```
[the remark this answers is not in the document]
Comment (veprbl) [reply pkgs/development/python-modules/fastparquet/default.nix]: Documentation says that python 3 is a requirement: https://fastparquet.readthedocs.io/en/latest/install.html

[the remark this answers is not in the document]
Comment (dotlambda) [reply pkgs/development/python-modules/fastparquet/default.nix]: Strange. Their setup.py explicitly mentions 2.7: https://github.com/dask/fastparquet/blob/1.1/setup.py#L21.

**Comment by veprbl** [comment]
@GrahamcOfBorg build python2Packages.fastparquet python3Packages.fastparquet

File: pkgs/development/python-modules/fastparquet/default.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (veprbl) [reply pkgs/development/python-modules/fastparquet/default.nix]: Good catch. Opened https://github.com/dask/fastparquet/pull/407

**Comment by veprbl** [comment]
@GrahamcOfBorg build python2Packages.fastparquet python3Packages.fastparquet

**Pull request merged**